src/teleop/gui/src/components/AutonTask.vue

```diff
@@ -37,7 +35,7 @@
         />
       </div>
     </div>
-    <div class="box1 data" :style="{ backgroundColor: nav_state_color }">
+    <div class="box box1" :class="[nav_state_color]">
```

Comment:
Also having a class named box1 is :vomiting_face: we should rename this whereever it is used.